lisa/transformers/dom0_kernel_installer.py

```diff
@@ -191,12 +238,105 @@ def install(self) -> str:
                 node.get_pure_path(f"/boot/config-{new_kernel}"),
             )
 
-        _update_mariner_config(
+        if mariner_version <= 2:
+            _update_mariner_config(
```

Comment:
Rename it to `_update_mariner_v2_config`

Reply:
But why isn't it called here for v3?

Reply:
V3 needs a separate config for permanent grub changes.

"dom0 source installer" was actually an older one which is not currently used. It was used only when our image was based on mariner 2, but now its mariner 3 and also, we dont really use the source installer. Probably similar changes like this PR is needed for dom0 source installer as well but since its not used at all I deprioritized it.

Reply:
So it's fine to rename the method, and we can skip the dom0 source installer this time. If the source installer shows a v2 config, people will understand that v3 isn't supported.